libs/shared/util-web-sockets/src/lib/use-status-web-sockets/use-status-web-sockets.ts

```diff
@@ -38,6 +38,7 @@ export function useStatusWebSockets() {
       environment: environment?.id,
       cluster: environment?.cluster_id,
       project: projectId,
+      version: versionId,
```

Comment:
`versionId` not used?

Reply:
Not yet but it will be when we'll refactor log pages.